releases/7.6.0.md

```diff
@@ -285,6 +285,58 @@ Minor incompatibilities:
         ```
         LOCALE_PATHS.insert(0, os.path.join(APP_ROOT, 'locale'))
         ```
+    4. Remove the `ARCHES_APPLICATIONS` setting. If your application is an Arches Application (designed for redistribution), you should create an AppConfig in apps.py with the class attribute `is_arches_application = True`.
```

Comment:
I'd argue that this should be included in `arches-templates` and `updateproject.py` , so that the assumption/paradigm continues that all Arches projects are apps, and the only difference is if they're declared in `INSTALLED_APPS` or not.